src/lib/y2storage/planned/can_be_formatted.rb

```diff
@@ -142,20 +141,27 @@ def setup_filesystem(filesystem)
         filesystem.label = label if label
         filesystem.uuid = uuid if uuid
         filesystem.mkfs_options = mkfs_options if mkfs_options
-        filesystem.mount_by = mount_by if mount_by
-        setup_fstab_options(filesystem)
+        setup_mount_point(filesystem)
+      end
+
+      def setup_mount_point(filesystem)
+        assign_mountpoint(filesystem)
```

Comment:
Consistency: assign_mountpoint() vs. xxx_mount_point() everywhere.

NP: And it's also `set_up_mount_point()`; "setup" is the noun. The verb is "to set up".
</wiseass_mode>  ;-)